### ISSUE TITLE
Install script no longer works on Tiger (stock TLS version too old for archive.org)

### ISSUE THREAD

**Issue by gsteemso** [mistydemeo/tigerbrew#948]
So, yeah, the install script can't do its thing.  I worked around it by using TenFourFox to download the Curl bottle from the URL given, manually mv'd it to the correct place in /tmp, then commented out the line that did the actual fetch attempt and re-ran the script, whereupon it finished up without apparent issues.

Alas, running "brew doctor" died upon fetching the Ruby bottle; the checksums don't match.  AGAIN.  Tigerbrew is rapidly becoming unusable because of this constant checksum-mismatch crap after every few packages I download.  What keeps CAUSING that?  A static file shouldn't produce a hash that varies over time!

**Comment by sevan** [comment]
One approach if you are starting with a fresh setup, you just need a clone of the Tigerbrew repo fetched on a more modern computer.
Copy that into place on your old Mac, and run bin/brew update (or install a desired package).
It will fetch the relevant parts correctly via HTTP (not HTTPS) and become ready for use.

I did look at updating the script in issue #936 but didn't follow it through as it needed more work to handle checksums and if you're going to need another computer to download the script, you might as well fetch the copy of the repo there instead (that's just my thinking).

**Comment by gsteemso** [comment]
While you are technically not wrong there, my main machine is the one running Tiger (it's a 2005 Power Mac G5).  Yeah, it's old, but in terms of hardware it's still amply capable -- but with the ongoing hassle of trying to work around Tigerbrew's weirdnesses, I'm looking at having to install a dual-boot configuration of Linux to keep it equally capable in software.

I looked at issue #936 but you didn't go into much detail -- what is involved beyond trying to force the Internet Archive to accept non -https access?

**Comment by sevan** [comment]
> While you are technically not wrong there, my main machine is the one running Tiger (it's a 2005 Power Mac G5). Yeah, it's old, but in terms of hardware it's still amply capable -- but with the ongoing hassle of trying to work around Tigerbrew's weirdnesses, I'm looking at having to install a dual-boot configuration of Linux to keep it equally capable in software.

Very cool :)
I think void linux has a powerpc build for Macs which is still maintained.
  
> I looked at issue #936 but you didn't go into much detail -- what is involved beyond trying to force the Internet Archive to accept non -https access?

How to perform SHA256 checksum, there isn't anything shipped with Tiger that can do that, if your start point is via the shell script.

Instead of trying to clone the repo, what if you downloaded a compressed archive of the repo from the github website. You mentioned you had TenFourFox.

**Comment by gsteemso** [comment]
I should mention that something is very odd with Tigerbrew's interactions with the git that it installs.  If I try to run `brew update`, well, the first time I did it of course the repository didn't yet exist, and the bit of the Ruby code for dealing with that scenario died messily.  I worked around that by inserting a manual `cd .git` and then carrying out the other activities in the script from within that directory.

I've no idea why, but in EVERY git interaction, Tigerbrew seems to keep deciding that /usr/local is the repository instead of /usr/local/.git, and when it correctly observes that it doesn't own /usr/local, it makes an enormous mess and deletes the .git directory entirely.  It seems a little suboptimal.  (I'm _still_ trying to make it do `brew update` even now that the repository has been populated.  I'll post back here if I get anywhere.)
